src/metadata/ActionMetadata.ts

```diff
@@ -256,7 +256,7 @@ export class ActionMetadata {
     
     static appendBaseRouteToRegexpRoute(route: RegExp, baseRoute: string) {
         if (!baseRoute || baseRoute === "") return route;
-        const fullPath = baseRoute.replace("\/", "\\\\/") + route.toString().substr(1);
+        const fullPath = `^${baseRoute}${route.toString().substr(1)}?$`;
```

Comment:
Could you describe more this stuff? Why you had to change it, maybe sample `route` and `baseRoute` values please.